malariagen_data/ag3.py

```diff
@@ -1004,7 +1018,9 @@ def snp_genotypes(
         debug = self._log.debug
 
         debug("normalise parameters")
-        sample_sets = self._prep_sample_sets_arg(sample_sets=sample_sets)
+        sample_sets = self._prep_sample_sets_arg(
+            sample_sets=sample_sets, sample_query=sample_query
```

Comment:
I'm not sure where `sample_query` is coming from here, it doesn't appear a parameter in `def species_calls()`